README.md

```diff
@@ -112,7 +113,8 @@ ARGUMENTS
 OPTIONS
   -h, --help     show CLI help
   -v, --version  show CLI version
-  --multi-app
+  --init         create the cdk directory before building the app and stack files
```

Comment:
I wonder if this should be the default behaviour for the `new` command? Perhaps we could consider that in a future PR though.